proteus/TwoPhaseFlow/TwoPhaseFlowProblem.py

```diff
@@ -79,9 +87,10 @@ def assert_initialConditions(self,ns_model,nd,initialConditions):
         assert 'vel_v' in initialConditions, 'Provide vel_v in ICs'
         if nd==3:
             assert 'vel_w' in initialConditions, 'Provide vel_w in ICs'
-        assert 'clsvof' in initialConditions, 'Provide clsvof in ICs'
-        if ns_model==1: #rans3p
-            assert 'pressure_increment' in initialConditions, 'Provide pressure_increment in ICs'
+        if ns_model == 1:
```

Comment:
Are you trying to make the use of `clsvof` to be linked with rans3p? and to use the other level set with rans2p? I am not opposed to that but I think in general it would be better to introduce a parameter called `ls_model` that is equal to 0 and 1 depending on which model you want to use (clsvof or ls+redist+vof+mcorr). If we do that then we have `ns_model` and `ls_model` so one can use either of the four possibilities. They all work with each other. After that we can decide what is set by default. What do you think?

Reply:
Yes, I think an `ls_model` parameter is a good idea and would be useful to choose and switch between the 2 models easily